CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 ### Added
 
 ### Changed
+- Changes to support CPU usage in Beats: opendbsd and freebsd return `ErrNotImplemented`
```

Comment:
This project is independent of Beats so I wouldn't mention Beats here at all. I recommend describing the change and how it affects the library or users of the library. E.g.

```
### Added
- Added method stubs to enable compilation for operating systems that are not 
supported by gosigar. All methods return `ErrNotImplemented` on these unsupported
operating systems.

### Changed
- On FreeBSD and OpenBSD the implementation of `ProcTime.Get` was changed to
return `ErrNotImplemented` instead of nil.
```

Reply:
You are right. I thought that the title of the PR must be included in the changelog.